### PR TITLE
if we panic in bolt/bbolt/dbd - DO NOT try to 'cleanly' unload wallet !

### PR DESCRIPTION
**Pull request by johnsonjh**

### CODE REVIEW

File: pktwallet/btcwallet.go [hunk 1]
```diff
@@ -126,7 +126,7 @@ func walletMain() er.R {
 	}
 
 	dbDir := networkDir(cfg.AppDataDir.Value, activeNet.Params)
-	loader := wallet.NewLoader(activeNet.Params, dbDir, cfg.Wallet, 250)
+	loader := wallet.NewLoader(activeNet.Params, dbDir, cfg.Wallet, 250) // XXX wtf this
```
Comment (cjdelisle) [inline pktwallet/btcwallet.go:129]: The 250 ?  That's how far back from the last known synced height the wallet will look to see if there are additional transactions, to double-check.

Comment (johnsonjh) [reply pktwallet/btcwallet.go:129]: > The 250 ? That's how far back from the last known synced height the wallet will look to see if there are additional transactions, to double-check.

gotcha :)

Comment (cjdelisle) [reply pktwallet/btcwallet.go:129]: Oops no, my mistake, actually it's about key derivation. The seed creates a stream of keys and the wallet needs to check if any of those keys were used, but it doesn't make sense to check thousands and thousands of keys if the user probably only used a few, so the behavior is to check the first 250 keys for any activity, and if any is spotted then check up to the key 250 past the last key to have been used.

Comment (johnsonjh) [reply pktwallet/btcwallet.go:129]: Yeah, leaving it was sort of an oversight on my side anyway, not really for public consumption. :innocent:

**Comment by johnsonjh** [comment]
There you are.

**Pull request merged**